.travis.yml

```
@@ -5,63 +5,85 @@ addons:
   apt:
     packages:
     - socat
-    - enchant
-    - aspell
-    - aspell-en
 
 env:
-  global:
-  - REDIS_TAGS="2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1" INSTALL_DIR=$HOME/redis
+- &ENV
```

This is a wrong syntax for global env. instead use:
```suggestion
  global:
    REDIS_TAGS: >-
      2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1
    INSTALL_DIR: >-
      $HOME/redis
    PYTEST_ADDOPTS: >-
      -n auto
```